### PR TITLE
Add initial Indonesian translation

### PR DESCRIPTION
**Pull request by zamronypj**

### CODE REVIEW

**Comment by sergeyklay** [comment]
@SidRoberts Could you please review?

File: en/reference/config.rst [hunk 1]
```diff
@@ -149,3 +149,38 @@ The above code produces the following:
     )
 
 There are more adapters available for this components in the `Phalcon Incubator <https://github.com/phalcon/incubator>`_
+
+Injecting Configuration Dependency
+----------------------------------
+You can inject configuration dependency to controller allowing us to use :doc:`Phalcon\\Config <../api/Phalcon_Config>` inside :doc:`Phalcon\\Controller <../api/Phalcon_Controller>`. To be able to do that, add following code inside your dependency injector script.
```
Comment (SidRoberts) [inline en/reference/config.rst]: Phalcon\Mvc\Controller

**Comment by SidRoberts** [comment]
Thank you. :) Is this ready to merge?

**Comment by zamronypj** [comment]
Only part of them are in Indonesian, but most of them is is still in English and in progress. If you think it is OK that most of translations are in English then you can merge it. I plan to translate it one page/ day.

**Comment by SidRoberts** [comment]
I'll merge this now and you can create a new PR for other pages. Thanks. :)

**Comment by sergeyklay** [comment]
@zamronypj Could you please drop Oracle documentation in in accordance with https://github.com/phalcon/cphalcon/issues/12008 and https://github.com/phalcon/cphalcon/pull/12009

**Comment by SidRoberts** [comment]
@sergeyklay @zamronypj Oracle references removed in f365b866ef95953d971de84862c7b47197bbe276.

**Comment by sergeyklay** [comment]
👍

**Comment by zamronypj** [comment]
Thanks @SidRoberts

**Pull request merged**